### PR TITLE
Support-for-JasminSMS

### PR DESCRIPTION
**Pull request by fynmanoj**

### CODE REVIEW

File: src/main/resources/db/migration/V1__Initial_Setup.sql [hunk 1]
```diff
@@ -47,7 +47,7 @@ CREATE TABLE m_outbound_messages (
   sms_bridge_id           BIGINT(20)                                      NOT NULL,
   mobile_number           VARCHAR(255)                                    NOT NULL,
   submitted_on_date       TIMESTAMP                                       NOT NULL,
-  delivered_on_date       TIMESTAMP                                       NOT NULL,
+  delivered_on_date       TIMESTAMP                                       NULL DEFAULT NULL,
```
Comment (avikganguly01) [inline src/main/resources/db/migration/V1__Initial_Setup.sql]: @fynmanoj : Don't change existing migration scripts.

**Review by avikganguly01 — CHANGES_REQUESTED** [review]
Add new migration script to account for updating delivered on date after receiving delivery status callback.

File: src/main/resources/db/migration/V1__Initial_Setup.sql (hunk 1 above)
Comment (fynmanoj) [reply src/main/resources/db/migration/V1__Initial_Setup.sql]: removed this change

**Comment by avikganguly01** [comment]
@fynmanoj Please update Readme on how to configure Jasmin.

**Pull request merged**